README.md

```diff
@@ -4,12 +4,11 @@ Sitio Web Asociación Blockchain de Costa Rica
 ## About Edenia
 
 <div align="center">
-	<a href="https://edenia.com">
-		<img src="https://raw.githubusercontent.com/edenia/.github/master/.github/workflows/images/edenia-logo.png" width="300">
-	</a>
+
+<a href="https://edenia.com"><img width="400" alt="image" src="https://raw.githubusercontent.com/edenia/.github/master/.github/workflows/images/edenia-logo.png"></img></a>
```

Comment:
alt="image" should actually be "edenia Logo" 

`alt` tag is a text that describes the image

https://blog.hubspot.com/marketing/image-alt-text

Reply:
fixed